docs/Contributing-back.rst

```diff
@@ -0,0 +1,610 @@
+to someone else in the EasyBuild community that may know the answer.
+
+The EasyBuild community is known to be very welcoming and helpful, and you too can be a part of that.
+
+
+.. _contributing_back_easyconfigs:
+
+Easyconfig contributions
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Please **contribute back easyconfig files** that are not included yet in
+(the ``develop`` branch of) the ``easybuild-easyconfigs`` GitHub repository at
+https://github.com/hpcugent/easybuild-easyconfigs.
+
+This includes easyconfigs for software that is not supported yet by EasyBuild,
+as well as updates to existing easyconfigs (e.g., version updates, using a
+different toolchain, etc.), even if you consider the updates to be trivial.
+Keep in mind that being able to use something that is known to work out-of-the-box
+can save quite a bit of time compared to having to tweak existing easyconfigs
+and validate the result installations.
+
+We highly recommend using the ``--new-pr`` and ``--update-pr`` command
+line options for contributing back easyconfig files; see :ref:`github_new_update_pr`.
+
+
+.. _contributing_back_code:
+
+Code contributions
+~~~~~~~~~~~~~~~~~~
+
+Of course you can also **contribute back code changes**, including bug fixes,
+enhancements or additional features in the EasyBuild framework, the easyblocks repository,
+the test suites or in related scripts.
+
+Do keep in mind that this requires some experience with Python,
+although you should be able to go a long way by using the existing code base as support.
+
+See :ref:`contributing_back_pull_requests` for more information on the practical
+aspects of contributing back code.
+
+
+.. _contributing_back_docs:
+
+Writing documentation
+~~~~~~~~~~~~~~~~~~~~~
+
+Another way to contribute back to EasyBuild without having to implement Python
+code is by **writing documentation**, i.e. enhancing or updating existing
+documentation or documenting features that are not covered yet.
+
+To contribute to the documentation hosted at http://easybuild.readthedocs.io/,
+you should open a pull request to the ``develop`` branch of the
+main EasyBuild repository at https://github.com/hpcugent/easybuild; see the ``docs`` subdirectory.
+
+A particularly useful tool is http://rst.ninjs.org/, which can be used to
+preview how the documentation written in ``reStructuredText`` format
+will be rendered on ``readthedocs.org`` (select the ``Nature`` theme for optimal
+results).
+
+
+.. _contributing_back_discuss:
+
+Joining the discussion
+~~~~~~~~~~~~~~~~~~~~~~
+
+Last but not least, you can actively join the discussions that arise on the EasyBuild
+mailing list, the ``#easybuild`` IRC channel and during the bi-weekly EasyBuild
+conference calls (see https://github.com/hpcugent/easybuild/wiki/Conference-calls).
+
+Engage with the EasyBuild community, and help steer EasyBuild development by
+participating in the often lively (yet civil and respectful) discussions regarding
```

Comment:
replaced `lively (yet civil and respectful) discussions` with `conversations` (which is a bit more boring, but fine ;))